Assets/Scripts/GameTime/Clock.cs

```diff
@@ -46,6 +45,7 @@ private void Update()
         /// </summary>
         private void UpdateTimeScale()
         {
+            _timeScale = 24 / (dayLength / 60); // calculating basic tick rate
```

Comment:
Könntest Du das vielleicht im Awake in nem Field `baseTimeScale` oder so speichern? Dann funktioniert es trotzdem und muss aber nicht jedes Mal neu berechnet werden